6-Final-Proj/server/tests/delete_todos.test.ts

```diff
@@ -0,0 +1,52 @@
+import { Testkit } from "./testkit";
+import { AxiosResponse, AxiosError } from "axios";
+import axios from "axios";
+import { TodoItem } from "../../common/types";
+import { getRandomItem, getRandomUserId } from "./utils";
+
+const testkit = new Testkit();
+
+describe("DELETE /todos", () => {
+  testkit.beforeAndAfterEach();
+
+  it("When the item exists, \
```

Comment:
when u run the tests, the output should be readable. look at the output and fix the describes and tests
for example.. tests should start with `should` b/c it output `it should...`